jupyter/rank_classification_using_BERT_on_Amazon_Review.ipynb

```diff
@@ -0,0 +1,498 @@
+    "            .build();\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Construct your model\n",
+    "\n",
+    "We will load our pretrained model and prepare the classification. First construct the `criteria` to specify where to load the embedding (DistiledBERT), then call `loadModel` to download that embedding with pre-trained weights. Since this model is built without classification layer, we need to add a classification layer to the end of the model and train it. After you are done modifying the block, set it back to model using `setBlock`.\n",
+    "\n",
+    "### Load the word embedding\n",
+    "\n",
+    "We will download our word embedding and load it to memory (this may take a while)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Criteria<NDList, NDList> criteria = Criteria.builder()\n",
+    "                .optApplication(Application.NLP.WORD_EMBEDDING)\n",
+    "                .setTypes(NDList.class, NDList.class)\n",
+    "                .optModelUrls(\n",
+    "                        \"https://alpha-djl-demos.s3.amazonaws.com/model/examples/distilbert.zip\")\n",
+    "                .optTranslator(new NoopTranslator(null)) // no preprocessing and postprocessing\n",
+    "                .optProgress(new ProgressBar())\n",
+    "                .build();\n",
+    "ZooModel<NDList, NDList> embedding = ModelZoo.loadModel(criteria);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create classification layers\n",
+    "\n",
+    "Then let's build a simple MLP layer to classify the ranks. We set the output of last FullyConnected (Linear) layer to 5 to get the predictions for star 1 to 5. Then all we need to do is to load the block into the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Block classifier = new SequentialBlock()\n",
+    "                .add(Linear.builder().setUnits(768).build()) // pre classifier\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Dropout.builder().optRate(0.2f).build())\n",
+    "                .add(Linear.builder().setUnits(5).build()) // 5 star rating\n",
+    "                .addSingleton(nd -> nd.get(\":,0\")); // Take [CLS] as the head\n",
+    "Model model = Model.newInstance(\"AmazonReviewRatingClassification\");\n",
+    "model.setBlock(classifier);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build DataManager\n",
+    "\n",
+    "`DataManager` is a class that is used primarily in the `Trainer` to manipulate data from the dataset to be appropriate for the current model. For our use case, we need to build a data manager that can do the following conversion:\n",
```

Comment:
Don't use the DataManager class. We are deprecating it for removal. You can replace it with either lambda blocks or forward with labels

Reply:
I changed to use lambda block now